db/src/db.rs

```diff
@@ -395,8 +396,9 @@ impl TypedSQLValue for TypedValue {
 /// Read the ident map materialized view from the given SQL store.
 pub fn read_ident_map(conn: &rusqlite::Connection) -> Result<IdentMap> {
     let mut stmt: rusqlite::Statement = conn.prepare("SELECT ident, entid FROM idents")?;
-    let m = stmt.query_and_then(&[], |row| -> Result<(String, Entid)> {
-        Ok((row.get(0), row.get(1)))
+    let m = stmt.query_and_then(&[], |row| -> Result<(symbols::NamespacedKeyword, Entid)> {
+        let ident: String = row.get(0);
+        Ok((to_namespaced_keyword(&ident).unwrap(), row.get(1)))
```

Comment:
Again, it's probably safest not to unwrap — we're already returning `Result`, so you could simply have `to_namespaced_keyword` return a `Result`, or make one with `.ok_or`.